File: C/Colleen/Colleen.c
#include <stdio.h>

// Outside comment

void print(char *s) { printf(s, 10, 10, 10, 10, 10, 10, 10, 10, 10, 34, s, 34, 10, 10, 10, 10); }

int main()
{
    // Inside comment
    char *s="#include <stdio.h>%c%c// Outside comment%c%cvoid print(char *s) { printf(s, 10, 10, 10, 10, 10, 10, 10, 10, 10, 34, s, 34, 10, 10, 10, 10); }%c%cint main()%c{%c    // Inside comment%c    char *s=%c%s%c;%c    print(s);%c    return 0;%c}%c";
    print(s);
    return 0;
}

File: Python/Colleen/Colleen.py
# Outside comment

def print_s(s):
	print(s)

if __name__ == "__main__":
	# Inside comment
	s = '# Outside comment\n\ndef print_s(s):\n	print()\n\nif __name__ == "__main__":\n	# Inside comment\n	s = ...\n	print_s(s)\n'
	print_s(s)
